### DISCUSSION TITLE
tab item customizing

### DISCUSSION THREAD

**Discussion by ryuhangyeong** [codesandbox/sandpack#705] [Q&A] [answered]
## Description

<img width="801" alt="image" src="https://user-images.githubusercontent.com/27342882/216750722-ea179272-40f2-4155-b515-b57166cbddb0.png">

I want to create a UI where I can put a delete button on each tab or add it to the right end of the tab. Is there any way to achieve this in sandpack?

`x` -> remove file
`+` -> create file

**Comment by ryuhangyeong** [comment]
## Self Answer

[Link](https://sandpack.codesandbox.io/docs/getting-started/custom-ui#tabs)

Found an option to delete the tab. But I haven't yet found a way to create additional buttons inside tabs or customize tabs!

**Comment by danilowoz** [comment, marked as answer]
Currently, there is no way to create additional buttons on the tabs. So, I recommend going to a custom solution meanwhile and building your own tab component, which will give you more flexibility in designing this functionality. 

That said, I'm curious if more people are interested in this feature. Could you create an issue for this feature request? 

Thanks!

**Reply by ryuhangyeong** [reply]
@danilowoz https://github.com/codesandbox/sandpack/discussions/709

Thanks for releasing such a wonderful library!

**Reply by danilowoz** [reply]
Moved it to an issue https://github.com/codesandbox/sandpack/issues/710
